config/routes.rb

```diff
@@ -1,14 +1,14 @@
 Rails.application.routes.draw do
-  # You can have the root of your site routed with "root"
-  # root 'welcome#index'
+  resources :profiles
+  resources :sessions
```

Comment:
Constrain this route to what you're using:
resources :sessions, only: [:new, :create, :destroy]